packages/core/src/amazonqGumby/chat/controller/messenger/messenger.ts

```diff
@@ -342,13 +345,13 @@ For more information, see the [Amazon Q documentation.](https://docs.aws.amazon.
         this.dispatcher.sendCommandMessage(new SendCommandMessage(message.command, message.tabId, message.eventId))
     }
 
-    public sendJobFinishedMessage(tabID: string, cancelled: boolean = false, jobStatus: string = '') {
+    public sendJobFinishedMessage(tabID: string, cancelled: boolean = false) {
         let message =
-            'I stopped your transformation. If you want to start another transformation, choose **Start a new transformation.**'
+            'I stopped your transformation. If you want to start another transformation, choose **Start a new transformation**.'
 
         if (!cancelled) {
             message =
-                'The transformation job has been completed. If you want to start another transformation, choose **Start a new transformation.**'
+                'The transformation job is over. If you want to start another transformation, choose **Start a new transformation**.'
```

Comment:
"has been completed" can be misleading since this message shows at the end of every job run attempt, successful or not.

Reply:
Question: Was this approved text changes. I'm okay with this text in the short term and we can follow up after.

Reply:
Let me check with Allie

Reply:
Update: I'm in the process of modifying all error message, so will leave this as it is for now, and I should have a PR out in near future to change a lot of the error handling / messaging.